controllers/datadogagent/feature/admissioncontroller/feature.go

```diff
@@ -57,6 +57,11 @@ func (f *admissionControllerFeature) Configure(dda *v2alpha1.DatadogAgent) (reqC
 		if ac.ServiceName != nil && *ac.ServiceName != "" {
 			f.serviceName = *ac.ServiceName
 		}
+		if ac.Registry != nil && *ac.Registry != "" {
```

Comment:
I think it could be helpful to add a comment above this line - something like, 
"set image registry from feature config or global config if defined"